api/include/opentelemetry/nostd/list.h

```diff
@@ -0,0 +1,254 @@
+#pragma once
+
+#include <algorithm>
+#include <array>
+#include <cstring>
+#include <iostream>
+#include <iterator>
+#include <memory>
+
+#include "opentelemetry/nostd/utility.h"
+#include "opentelemetry/version.h"
+
+OPENTELEMETRY_BEGIN_NAMESPACE
+namespace nostd
+{
+// Non-standard implementation of the list data structure.
+template <class T>
+class list
```

Comment:
Naming this `vector` would  be more accurate.

Reply:
Changed the name to vector.